docs/01-overview/02-main-areas/service-management/smgt-13-hb-test-addons.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Test addons
+---
+
+If you want to test whether your addons are correct, you can use the dry run mode to check the generated manifests of the chart without installing it.
+
+The `--debug` option prints the generated manifests. As a prerequisite, you must install [Helm](https://github.com/kubernetes/helm) on your machine to run this command:
```

Comment:
Does the prerequisite apply to the dry run mode too, or only to the --debug option?